specification/sphere/resource-manager/Microsoft.AzureSphere/stable/2024-04-01/azuresphere.json

```diff
@@ -620,13 +620,14 @@
             "type": "string"
           },
           {
-            "name": "proofOfPossessionNonceRequest",
+            "name": "body",
```

Comment:
this is a SDK change. You want to use `@encodedName("application/json", proofOfPossessionNonceRequest)` to keep it.

Reply:
`"x-ms-client-name": "proofOfPossessionNonceRequest"` to rename the name in sdk.